src/pages/iou/IOUDetailsModal.js

```diff
@@ -0,0 +1,159 @@
+import React, {Component} from 'react';
+import {View, ActivityIndicator, ScrollView} from 'react-native';
+import PropTypes from 'prop-types';
+import {withOnyx} from 'react-native-onyx';
+import lodashGet from 'lodash/get';
+import _ from 'underscore';
+import styles from '../../styles/styles';
+import ONYXKEYS from '../../ONYXKEYS';
+import themeColors from '../../styles/themes/default';
+import HeaderWithCloseButton from '../../components/HeaderWithCloseButton';
+import Navigation from '../../libs/Navigation/Navigation';
+import ButtonWithLoader from '../../components/ButtonWithLoader';
+import ScreenWrapper from '../../components/ScreenWrapper';
+import {settleIOUReport} from '../../libs/actions/IOU';
+import {fetchIOUReportByID} from '../../libs/actions/Report';
+import ReportActionItemIOUPreview from '../../components/ReportActionItemIOUPreview';
+import iouTansactionPropTypes from './iouTansactionPropTypes';
+import IOUTransactions from './IOUTransactions';
+import withLocalize, {withLocalizePropTypes} from '../../components/withLocalize';
+import compose from '../../libs/compose';
+
+const defaultProps = {
+    iou: {},
+};
+
+const propTypes = {
+    // URL Route params
+    route: PropTypes.shape({
+        // Params from the URL path
+        params: PropTypes.shape({
+            // chatReportID passed via route /iou/:chatReportID
```

Comment:
Are the links in these comments correct? I thought it was `/iou/details/:chatReportID/:iouReportID`

Reply:
Ah good spot, they were changed back.